signcrypt_seal.go

```diff
@@ -69,6 +69,7 @@ func (sss *signcryptSealStream) signcryptBytes(b []byte) error {
 
 	nonce := nonceForChunkSigncryption(sss.numBlocks)
 
+	// TODO: Encode payload chunk.
```

Comment:
Not that I know of, which part did you mean? We double-encode the header (list of objects -> msgpack `array` -> msgpack `bin`), but I don't think we do that anywhere in the payload.